service-manual/measurement/completion-rate.md

```diff
@@ -20,117 +20,85 @@ breadcrumbs:
 
-Measuring end-to-end completion rates helps to identify whether users have problems completing a transaction. Analysing the drop-out rate at each step of a transaction will pinpoint the specific processes that users fail to complete.
-
-## What you will be measuring
-
-The completion rate measures the proportion of people who start a transaction and are able to complete it.
-
-A transaction is a self-contained process that's defined by the service manager, and is typically only completed when a user has been through the whole thing end to end.
-
-A transaction completed only partially online means the user may complete a single process within the overall transaction, like booking an appointment or completing a part of an application.
+Once your service goes live you’ll find steps that cause users to give up, and bugs that make the service impossible to complete. And as you fix these issues, your completion rate will rise.
 
 ## How to measure completion rate
 
-1.    Work out the number of completed transactions.
-2.    Divide it by the number of started transactions, expressed as a percentage.
-
-### Before the transaction starts
+1. Count the number of completed transactions
+2. Divide it by the number of started transactions, as a percentage
 
-Before the transaction has started, users should clearly be told:
+## What is a completed transaction?
 
-* what the outcome of the transaction will be
-* who can use it
-* how long it's likely to take
-* what they’ll need to complete it (eg a reference number or credit card)
+A transaction is completed when **the service did what it was supposed to do**.
 
-Also provide users with the eligibility criteria and the costs to complete the transaction. This will help you to reduce dropouts later on in the transaction.
+Often this will be because the user was successful in achieving their desired outcome, for example:
 
-Typically the eligibility criteria will be given on a single page. In some cases, there may be a set of pages that checks a user’s eligibility based on information they provide.
+* A user claimed Carer’s Allowance
+* A user paid their vehicle tax
 
-Don’t make it possible for users to bypass your service’s start page via links or search engine results. Users who try to directly access another page in the transaction should be sent back to your start page, unless they are resuming a previously saved transaction.
+Sometimes, even though **the service did what it was supposed to do**, the user won’t be successful. These still count as completed transactions. For example:
 
-### Start and end points
+* A user couldn’t claim Carer’s Allowance because they weren’t eligible
+* A user couldn’t pay their vehicle tax because they have already paid
 
-Transactions will begin and end on GOV.UK to allow GDS to monitor completion rates. 
+In all these cases, the user will reach a [transaction end page](https://designpatterns.hackpad.com/Transaction-end-pages-xkOPGx6R1iM).
```

Comment:
Sorry - it's a regex. In essence, replace `all these` with `all of these`, because it reads better.